intel_nn_hal/GnaPreparedModel.cpp

```diff
@@ -2529,11 +2523,21 @@ BaseOp* GnaPreparedModel::operationQuantize(const Operation& operation, bool dum
             break;
     }
 
-    if (static_cast<int>(operandDetails.lifetime) == (int)V1_3_OperandLifeTime::SUBGRAPH_INPUT) {
+    if (static_cast<int>(operandDetails.lifetime) == static_cast<int>(OperandLifeTime::CONSTANT_COPY)) {
```

Comment:
It's already 4 levels of `if else` blocks.. wanna use `switch case`?

Reply:
Makes sense